content/2017-munich/register.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Register
+---
+
+## Register to attend
+
+Attendee registration will open soon — fill out the form linked below to get
+notified when the registration opens:
+
+<a class="btn btn-lg btn-default" href="https://docs.google.com/forms/d/12H6o3vvnnQUJSDV1mQ77EqpU9B64Jk70iFboHwb2y3M/viewform" target="_blank" role="button"><i class="fa fa-briefcase"></i>&nbsp;&nbsp;Register</a>
```

Comment:
That's still the old form (it says 2016) - did you intend to link a new one already?